src/components/HeatMap/Cell/Cell.styles.js

```diff
@@ -6,12 +6,12 @@ const Wrapper = styled.div`
   display: flex;
   align-items: center;
   justify-content: center;
-  width: 40px;
-  height: 40px;
+  height: ${({ theme }) => theme.heatmap.cellHeight};
+  width: ${({ theme }) => theme.heatmap.cellWidth};
   background-color: ${(props) => (
     props.children < 10
-      ? props.theme.heatMapColors[`color${props.children}`]
-      : props.theme.heatMapColors.color10
+      ? props.theme.heatmap.colors[props.children]
+      : props.theme.heatmap.colors['10']
```

Comment:
`props.theme.heatmap.colors[10]` works fine as well, the `10` doesn't need to be a string

Reply:
In fact the key in the theme is not a string but a number, right? So let's make it a number here as well.